python/paddle/distributed/auto_parallel/tuner/config.py

```diff
@@ -0,0 +1,135 @@
+#   Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
```

Comment:
We should unified this configuration process for auto parallel. For now this is only used by the optimization tuner. I suggest to move this out of the tuner folder to the auto_parallel folder and improve this to handle all the configuration related to auto parallel (including the adapter from Fleet strategy to auto parallel configuiration).

Reply:
yes，here will be re-design。